examples/grpc.core/Examples.GrpcCore.AspNetCore/Controllers/WeatherForecastController.cs

```diff
@@ -7,12 +7,12 @@ namespace Examples.GrpcCore.AspNetCore.Controllers;
 
 [ApiController]
 [Route("[controller]")]
-public class WeatherForecastController : ControllerBase
+internal class WeatherForecastController : ControllerBase
```

Comment:
I just tested this. AspNetCore won't auto-register `internal` controller so this effectively breaks the app. Curious, what is the warning you are getting spurring this change?

Reply:
`dotnet format opentelemetry-dotnet-contrib.sln` when executing in .NET9 SDK context (I just remove global.json file on my env).